sycl/test/basic_tests/buffer/buffer_interop.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out -L %opencl_libs_dir -lOpenCL
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out -I %sycl_include -L %opencl_libs_dir -lOpenCL
```

Comment:
Why `-I %sycl_include` is added to the command line which contains `-fsycl` flag? Does it mean that user should add path to headers even if our compiler is invoked?

Reply:
it was done by a mistake.
removed